standard/statements.md

```diff
@@ -875,12 +875,12 @@ The *statement_list*s of a *switch_block* may contain declaration statements ([
 A switch label is reachable if at least one of the following is true:
 
 - The switch expression is a constant value and either
-  - the label is a `case` whose pattern *would match* (XREF to “would match” in patterns.md) that value, and label’s guard is either absent or not a constant expression with the value false; or
+  - the label is a `case` whose pattern *would match* that value, and label’s guard is either absent or not a constant expression with the value false; or
```

Comment:
Xref to 11.4?

Reply:
I was wondering about the new pattern forms general clause, where "match" is defined.